mozdownload/cli.py

```diff
@@ -163,7 +166,7 @@ def cli(argv=None):
             scraper_type = 'direct'
 
         build = factory.FactoryScraper(scraper_type, **kwargs)
-        build.download()
+        build.download(kwargs['print_url'])
```

Comment:
We should not call download if we only want to print the URL. Add the if condition here instead.